### PR TITLE
Use bug type for tag

### PR DESCRIPTION
**Pull request by pcrov**
I vaguely remember someone wanting this.

Also was getting errors with the existing table selection code that apparently live jeeves isn't? I hope the html we're fed isn't so dissimilar that the query I used instead breaks.

### CODE REVIEW

File: src/Plugins/PHPBugs.php [hunk 1]
```diff
@@ -89,7 +89,8 @@ private function checkBugs()
 
             foreach ($this->rooms as $room) {
                 yield $this->chatClient->postMessage($room, \sprintf(
-                    "[tag:php] [tag:bug] %s – [#%d](%s)",
+                    "[tag:php] [tag:%s] %s – [#%d](%s)",
+                    $bug["type"],
```
Comment (DaveRandom) [inline src/Plugins/PHPBugs.php]: What is the range of possible values for this?

Comment (pcrov) [reply src/Plugins/PHPBugs.php]: Bug, Doc, Req, and SecBug

Comment (kelunik) [reply src/Plugins/PHPBugs.php]: Could you map that to `bug`, `documentation`, `feature-request`, `security`?

**Comment by DaveRandom** [comment]
Merging for testing in production because #yolo

**Pull request merged**